src/ol/control/fullscreencontrol.js

```diff
@@ -37,13 +36,33 @@ ol.control.FullScreen = function(opt_options) {
   this.cssClassName_ = goog.isDef(options.className) ?
       options.className : 'ol-full-screen';
 
+  var label = goog.isDef(options.label) ?
+      options.label : '\u2194';
+
+  /**
+   * @private
+   * @type {Node}
+   */
+  this.labelNode_ = /** @type {Node} */ (goog.isString(label) ?
+          goog.dom.createTextNode(label) : label);
```

Comment:
In the case where a `Node` is given, like in the above example. I should change the type definition of `label` to `{string|Node|undefined}` (for all controls).